functions.py

```diff
@@ -2,3 +2,35 @@
 Write a simple script that demonstrate your understanding of functions in python 
 '''
 
+def printme( str):
+    "this print a passed string into this function"
+    print (str)
+    return
+
+    printme("I am first call to user defined function")
```

Comment:
Are these going to run? 
It looks to me like they are part of printme(str) function

Reply:
they are running